docs/annotations.md

```diff
@@ -0,0 +1,19 @@
+# Annotations
+
+## `metalkast.io/redfish-urls`
+
+This annotation is specified on `Secret` objects to allow `kast generate` command
+to discover and generate `BareMetalHost` objects.
+
+URLs are specified one per line. Secret should contain `username` and `password`
+which are credentials to the specified Redfish URLs.
+
+## `metalkast.io/bootstrap-cluster-apply`
+
+`kast bootstrap` applies all the given manifests to two different clusters: bootstrap and target.
+Bootstrap cluster is temporary and is started on one of the baremetal nodes. The purpose of this cluster is to initialize Cluster API. Since this cluster is not managed by Cluster API, all the objects are later moved to target cluster and the bootstrap cluster is destroyed.
+
+Set the value of this annotation to `"false"` to any manifests that you don't want to be applied to the bootstrap cluster. Typical scenarios for this would be:
```

Comment:
We should consider switching it to "false" by default and only setting it to "true" for resources that should be deployed to bootstrap cluster - we will likely have a lot in "default" template, and only some part of it required to be on the bootstrap cluster.